cmd/admin.go

```diff
@@ -57,8 +58,55 @@ to make automatic initialization process more smoothly`,
 			},
 		},
 	}
+
+	subcmdChangePassword = cli.Command{
+		Name:   "change-password",
+		Usage:  "Print encoded version of a given password",
```

Comment:
I meant, that the encoded password isn't being printed. Should the usage not just be "change password of a user" ?